packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -1277,23 +1277,29 @@ class ServiceMap extends ServiceObject implements Map<String, dynamic> {
   @override
   String toString() => _map.toString();
   @override
+  // ignore: undefined_method, override_on_non_overriding_method, non_type_as_type_argument
```

Comment:
Should we add a TODO for removing these again? Assuming they are just temporarily?